lib/create-command-server.js

```diff
@@ -70,27 +58,56 @@ const onConnection = websocket => {
   });
 };
 
+class CommandServer extends WebSocketServer {
+  /**
+   * Broadcast message to all clients.
+   * @param message
+   */
+  broadcast(message) {
+    const packedMessage = JSON.stringify(message);
+    /** @type {Set<WebSocketWithData>} */ (this.clients).forEach(websocket => {
```

Comment:
This kind of makes me feel like we should be using a WeakMap to associate the session ID. Not in this patch, of course.

Reply:
Yeah that sounds like a good idea. Adding post-hoc properties like `sessionId` makes it hard to track with type analysis where it is assigned or used.